PJ_SIM_SUNBNB_2023-25 [FE] User Profile

ThereseRoca opened the pull request:
# Issue Link
[PJ_SIM_SUNBNB_2023-25 [FE] User Profile](https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-25)

# Slack Link
https://sun-philippine.slack.com/archives/C05GURS7N3U/p1691147422055179

# Definition of Done
Create a markup for the user profiles with the components:
- [x] Full Name
- [x] User Image
- [x] Phone Number
- [x] Description
- [ ] List of Owned Listings (if Host)
- [x] List of Made Reviews (if Client)

# To Do
- Show owned listings of the user if he is a host when the listing item component is merged

# Expected Output
Host
![image](https://github.com/framgia/pj_sim-sunbnb/assets/138637515/d9fd4c5c-8234-4633-854a-422d7cfd54b3)

Client
![image](https://github.com/framgia/pj_sim-sunbnb/assets/138637515/0ee6c3bc-0ef2-45db-b013-48dd08da543d)

# Screenshots

Client View
![image](https://github.com/framgia/pj_sim-sunbnb/assets/137018536/68a50260-1f4f-4700-bae8-3a21f5df18f7)

Host View
![image](https://github.com/framgia/pj_sim-sunbnb/assets/137018536/b815e678-9dee-4a29-9d15-2d8e5fa441c0)

roseaugusto commented on fe/src/App.vue:
```diff
@@ -15,4 +15,6 @@ export default {
     NavBar,
   },
 };
+
```
is this space necessary?

ThereseRoca replied:
I removed the extra space already po ma'am.

roseaugusto commented on fe/src/components/Navbar/NavbarProfile.vue:
```diff
@@ -54,6 +54,9 @@ export default {
     viewProfile() {
       // Implement action to view profile
+      this.$router.push('/ClientViewProfile')
+
```
remove extra spaces

ThereseRoca replied:
I removed the extra space already po ma'am.

roseaugusto commented on fe/src/pages/ClientViewProfile.vue:
```diff
@@ -0,0 +1,68 @@
+</script>
+
+<style scoped>
+
```
remove unnecessary spaces here same with the bottom.

ThereseRoca replied:
I removed the extra space already po ma'am.

ThereseRoca committed:
PJ_SIM_SUNBNB_2023-25 [FE] User Profile

Show Profiles

Fix issues

Fix comments and spaces

Add changes

Fix small issues

Add Host Tab in Host Profile

Remove tab component from profile

Add changes to HostTab

roseaugusto commented on fe/src/components/Tab/HostTab.vue:
```diff
@@ -1,47 +1,53 @@
-    switch(option){
-      case "list": isActive = true; 
+function changeActive(option) {
+  switch (option) {
```
is parameter `option` only has two type? if yes, please change to boolean and you can change name to `isListing` or `isReservation`

andreenriquez11 replied:
Changed it already maam
